include/experimental/__p1673_bits/blas2_matrix_rank_2_update.hpp

```diff
@@ -48,6 +48,55 @@ namespace experimental {
 inline namespace __p1673_version_0 {
 namespace linalg {
 
+namespace {
+
+template <class Exec, class x_t, class y_t, class A_t, class Tr_t, class = void>
+struct is_custom_symmetric_matrix_rank_2_update_avail : std::false_type {};
+
+template <class Exec, class x_t, class y_t, class A_t, class Tr_t>
+struct is_custom_symmetric_matrix_rank_2_update_avail<
+  Exec, x_t, y_t, A_t, Tr_t,
+  std::void_t<
```

Comment:
why do we want all to inherit from true/false_type?

Reply:
@fnrizzi It would be weird if `Tpetra::Vector` only inherited from `Tpetra::DistObject` for certain Scalar types but not others, right?  Similarly, it would be weird if only some specializations of a traits class inherited from `std::integral_constant<bool, value>`, but not others.

Reply:
ah in that sense yes, i agree! I was misunderstanding you. 
if I understand correctly, what you mean is that the customization trait inherits from `false_type` when it is false, but when is true it just has `value = true` in it. So we need to fix this inconsistency, right?

Reply:
@mhoemmen what if we leave the "true" case as is, and we just use `value = false` instead of inheriting from `false_type`?